cmd/tendermint/commands/reset_priv_validator.go

```diff
@@ -62,11 +79,71 @@ func resetAll(dbDir, privValKeyFile, privValStateFile string, logger log.Logger,
 	} else {
 		logger.Error("error removing all blockchain history", "dir", dbDir, "err", err)
 	}
-	// recreate the dbDir since the privVal state needs to live there
+
+	return resetFilePV(privValKeyFile, privValStateFile, logger, keyType)
+}
+
+// resetState removes address book files plus all data.
+// Exported so other CLI tools can use it.
```

Comment:
It's not exported—I'm not sure whether it's the comment or the function that wants to be fixed.

Reply:
removed it thank you. There is no need to export.